### ISSUE TITLE
Loading mcclim: Clasp currently can't handle anything but base-char in input streams

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#651]
Small reproducible test case:
```common-lisp
(with-input-from-string (stream (make-string 4 :initial-element (code-char 256)))
           (read-line stream))
Condition of type: SIMPLE-PROGRAM-ERROR
Clasp currently can't handle anything but base-char in input streams
````

**Comment by kpoeck** [comment]
The limitation is clearly documented in the code
```c++
CL_DEFUN T_mv cl__read_line(T_sp sin, T_sp eof_error_p, T_sp eof_value, T_sp recursive_p) {
....
    if (!clasp_base_char_p(cc)) {
        SIMPLE_ERROR(BF("Clasp currently can't handle anything but base-char in input streams"));
````
I asssume if a non-base-char is found, a StrWNs_sp needs to be used as buffer and returned

**Comment by kpoeck** [comment]
In Clim this errors in parse-fontconfig-output in fontconfig.lisp. I put a handler-case around the read-line to silence to error (the particular line with the wide-character does not seem to be important

**Comment by kpoeck** [comment]
Fixed with #653
